sites/public/pages/applications/start/choose-language.tsx

```diff
@@ -136,7 +136,7 @@ const ApplicationChooseLanguage = () => {
 
                 {listing.applicationConfig.languages.map((lang, index) => (
                   <Button
-                    className="language-select mx-1"
+                    className="language-select mx-1 mb-2 sm:mb-0"
```

Comment:
Can you update this to use the `md` selector which we rely on more? (I think the whole repo has 100+ instances of `md` and three of `sm`)